phoenix5-spark3/README.md

```diff
@@ -39,7 +80,9 @@ UPSERT INTO TABLE1 (ID, COL1) VALUES (2, 'test_row_2');
 ```
 
 ### Load as a DataFrame using the DataSourceV2 API
+
 Scala example:
+
```

Comment:
I know you didn't touch that part, but do we still need the SparkContext import ?